lib/address.ml

```diff
@@ -0,0 +1,23 @@
+open Bin_prot.Std
+open Ppx_compare_lib.Builtin
+open Ppx_hash_lib.Std
+open Hash.Builtin
+open Ppx_sexp_conv_lib.Conv
+
+type t = {
+  address : string;
+  port : int;
+}
+[@@deriving bin_io, eq, compare, hash, sexp, show { with_path = false }]
+
+let create address port = { address; port }
```

Comment:
The conventional babe for these functions in OCaml is `make`

Reply:
Thanks, done.

Reply:
IIRC, `make` is for functors (e.g. Map.Make, Set.Make) and `create` is more common for e.g. `Queue.create`, `Bytes.create`, etc. Not going to block on this though 👍